store/src/flat_block_body.rs

```diff
@@ -137,17 +137,13 @@ pub(crate) fn deserialize_transaction(
         outputs.push(output);
     }
     let witnesses: Vec<Witness> = config.deserialize(&tx[header[5]..])?;
-    let hash: H256 = config.deserialize(&tx[header[6]..])?;
-    let witness_hash: H256 = config.deserialize(&tx[header[7]..])?;
-    unsafe {
-        Ok(TransactionBuilder::default()
-            .version(version)
-            .deps(deps)
-            .inputs(inputs)
-            .outputs(outputs)
-            .witnesses(witnesses)
-            .build_unchecked(hash, witness_hash))
-    }
+    Ok(TransactionBuilder::default()
```

Comment:
We wrote `hashes` into storage as caches. `build(..)` always computes those hashes again.

Reply:
updated.